odoo/tests/common.py

```diff
@@ -701,6 +701,8 @@ def setUpClass(cls):
         cls.env = api.Environment(cls.cr, odoo.SUPERUSER_ID, {})
         cls.addClassCleanup(cls.env.reset)
 
+        cls.env['res.currency'].with_context({'active_test': False}).search([('name', 'in', ['EUR', 'USD'])]).write({'active': True})
```

Comment:
I wonder if the tests that use these currency should not activate them manually.

`TestStockValuationLayerRevaluation` is an example
`TestHasGroup` above is also modified since the activation of these currency will imply an additional group on the internal user group.

Reply:
IMHO this has nothing to do in test/common.
-> Test common shouldn't be aware of odoo buisness.
-> This query will be executed for EVERY TEST. This will add tons of useless query on runbot.

Reply:
OK I've separated it to the individual tests thats require it ... see 982e29dfa45ac4f7331086781b1a3235c84cf743
better?

Reply:
Yes tanks!